backend/doc/setting.md

```diff
@@ -192,6 +192,14 @@ For instance:
 
 The last one will check the internal user database first. Then, if the auth fails, it will also try to authenticate the user using LDAP.
 
+The authentication can be done with a login, a login can be either email or username of user. In case there is
+no clear distinction in the authentication method used, login with "@" will be considered as email and login without as username.
```

Comment:
Is this possible that there is an ambiguity though? `@` is forbidden in the username. I would drop this sentence entirely.

Reply:
The Api auth method does make distinction between email and username field, other auth method that require login use "login" which will check either email or username depending if "@" is provided.

Reply:
Ah, ok. I didn't understand this sentence in this sense. I'll add a suggestion